### PR TITLE
Leaderboard Add style

### PR DESCRIPTION
**Pull request by diegoh40**
In this final part, I have customized the style of my page and improved some functionalities on the optional suggestions passed from the code checker that I found pertinent.
image

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -20,20 +20,20 @@ h1 {
...
-  background-color: #f2f2f2;
+  background-color: rgb(127, 149, 3);
 }
 
 #scores tr:hover {
-  background-color: #ddd;
+  background-color: rgb(127, 149, 3);
 }
 
 .refresh {
```
Comment (Mihndim2020) [inline src/style.css:39]: - [ ] As it is a requirement for this project, " `App is properly styled and its layout looks like the wireframe` ", please make the components of your app are layed out as in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). The refresh and the submit buttons are right aligned, please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @diegoh40, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments below and under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 #### I will leave the comments here, since your `Readme file` is not accessible for this `PR` as you did not make any change on it ####

- [ ] Please kindly give a meaningful title to your project in the readme file, replace the placeholder text `Project Name` with the title your project. This will make your readme file to look more professional. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by diegoh40** [comment]
As suggested by the code reviewer I improved the placement of the refresh and submit buttons and added an appropriate title to my README,

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @diegoh40, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean/Organized code
:heavy_check_mark: Good job with UI

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**